GTG/gtk/data/main_window.ui

```diff
@@ -156,7 +156,7 @@
             <property name="can_focus">False</property>
             <property name="receives_default">True</property>
             <property name="action_name">win.start_custom</property>
-            <property name="text" translatable="yes">Pick a date...</property>
+            <property name="text" translatable="yes">Pick a Date...</property>
```

Comment:
Unicode typography for ellipsis - https://developer.gnome.org/hig/stable/typography.html.en